### PR TITLE
TM4 interface fixes

### PR DESCRIPTION
**Pull request by kamicut**
- Fixes prefixes for the TM4 interface
- Adds TM4 to select in admin page
- Fix tests
- Filters out draft campaigns from frontpage map

### CODE REVIEW

File: api/src/db/seeds/test/campaigns.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ exports.seed = async (knex) => {
     dirname: path.join(__dirname, '..', '..', '..', '..', 'tests', 'tapes')
   }))
 
-  const tm4proxy = http.createServer(yakbak('https://tasks-backend.openstreetmap.us/', {
+  const tm4proxy = http.createServer(yakbak('https://tasks-backend.openstreetmap.us/api', {
```
Comment (LanesGood) [inline api/src/db/seeds/test/campaigns.js:16]: @kamicut I see a response for the original URL, but a 404 for this change.

Comment (kamicut) [reply api/src/db/seeds/test/campaigns.js:16]: Yeah, it's only used in the construction of API urls as a prefix, for example https://tasks-backend.openstreetmap.us/api/v2/projects

**Comment by LanesGood** [comment]
@kamicut in local development, I can run an instance of Scoreboard with 550 campaigns, and only one user. All the campaigns are MapRoulette. I've gone through the steps of yarn seed and migrate, but seed fails as usual for me and migrate doesn't throw any errors.

**Review by LanesGood — APPROVED** [review]
@kamicut thanks, let's merge it in! 🚀

**Pull request merged**